tests/unittests/objects/report/test_report_with_gitleaks.py

```diff
@@ -0,0 +1,154 @@
+#
```

Comment:
I'm seeing this message in some new files, but not all of them. Can you add this to the top of new files?

Reply:
I found a lot more of these in the test directory. I went ahead and added comments to the files that were already present as well.

Reply:
Thank you!